config/crds/knative_service.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 The Knative Authors
```

Comment:
this is needed for testing to work and `controller-runtime` testing library requires to register the crds, I tried pointing to the vendor knative directory but it did not work because there are other manifests which are not crds. I can ask on kubebuilder chat if anyway to avoid doing this.  
`CRDDirectoryPaths: []string{filepath.Join("..", "..", "..", "..", "config", "crds")},`

Reply:
discussed this issue on kubebuilder slack channel and they asked to file a ticket to `controller-runtime` (https://github.com/kubernetes-sigs/controller-runtime/issues/394). They have suggested a walk-around to pass a slice of parsed CRD objects, will investigate and address this in a separate PR.